### PR TITLE
Fix toggle icons

### PR DESCRIPTION
**Pull request by davidrapson**
I removed this by mistake :(

@afiore @tudorraul

### CODE REVIEW

File: frontend/assets/javascripts/src/modules/toggle.js [hunk 1]
```diff
@@ -33,6 +34,15 @@ define(['$', 'bean'], function ($, bean) {
         }
     };
 
+    var toggleIcon = function($elem) {
+        var iconName = $elem.data(TOGGLE_DATA_ICON);
+        var iconRef = $elem[0].querySelector('svg > use');
+        if (iconName && iconRef) {
+            $elem.data(TOGGLE_DATA_ICON, iconRef.getAttribute('xlink:href').replace('#icon-', ''));
```
Comment (afiore) [inline frontend/assets/javascripts/src/modules/toggle.js]: Maybe it would be worth moving the '#icon' into a constant

**Comment by davidrapson** [comment]
@afiore OK otherwise?

File: frontend/assets/javascripts/src/modules/toggle.js (hunk 1 above)
Comment (davidrapson) [inline frontend/assets/javascripts/src/modules/toggle.js]: @afiore OK otherwise?

Comment (afiore) [inline frontend/assets/javascripts/src/modules/toggle.js]: :+1: ok otherwise

**Pull request merged**